cmscontrib/loaders/tps.py

```diff
@@ -205,6 +205,7 @@ def get_task(self, get_statement=True):
 
         args['min_submission_interval'] = make_timedelta(60)
         args['min_user_test_interval'] = make_timedelta(60)
+        args['restricted_time'] = make_timedelta(60*15)
```

Comment:
This would mean the restriction is only enforced for 15 minutes? Perhaps it should be
```suggestion
        args['restricted_time'] = make_timedelta(-60*15)
```